package.json

```diff
@@ -1,28 +1,35 @@
 {
-	"name": "loggly-interface",
-	"description": "An module for pushing logs to Loggly",
-	"version": "0.2.1",
-	"homepage": "https://bitbucket.org/hxshortbreaks/loggly-interface",
-	"author": {
-		"name": "Shortbreaks",
-		"email": "shortbreaksdev@holidayextras.com"
-	},
-	"repository": {
-		"type": "git",
-		"url": "git@bitbucket.org:hxshortbreaks/loggly-interface.git"
-	},
-	"license": "MIT",
-	"main": "index",
-	"engines": {
-		"node": ">= 0.10.0"
-	},
-	"scripts": {
-		"start": "node launch.js"
-	},
-	"dependencies": {},
-	"devDependencies": {
-		"grunt": "~0.4.4",
-		"grunt-contrib-jshint": "^0.10.0",
-		"grunt-jscs": "^0.6.2"
-	}
+  "name": "loggly-interface",
+  "description": "An module for pushing logs to Loggly",
+  "version": "0.3.0",
+  "homepage": "https://github.com/holidayextras/loggly-interface",
+  "author": {
+    "name": "Shortbreaks",
+    "email": "shortbreaksdev@holidayextras.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:holidayextras/loggly-interface.git"
+  },
+  "license": "MIT",
+  "main": "index",
+  "engines": {
+    "node": ">= 0.10.0"
+  },
+  "scripts": {
+    "start": "node launch.js",
+    "test": "istanbul cover _mocha -- --recursive test/ -R spec"
```

Comment:
Should this be - ` grunt jshint && grunt jscs && istanbul cover _mocha -- test/ -R spec `